about.html

```diff
@@ -0,0 +1,170 @@
+        <p>
+          The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years. This year, we hope to expand our invitation list including organizations and invidivuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge. So if you're active and engaged in the worlds of open content and knowledge - free software advocates, Wikipedians. Open Knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together.
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions
+          about Student StartUp Lab Summit 2015!
+        </p>
+        <p class="email-address">
+          info@startuplab.com
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="flex-content">
+      <h2> Student StartUp Lab Global Summit 2021 Logo</h2>
+      <hr>
+      <p class="mobile-para">The logo of CC Global Summit 2015 was decided through the logo competition from 8 June to 7 July.</p>
+      <p class='desktop-para'> The logo of CC Global Summit 2015 was decided through the logo competition from 8 June to 7 July.
+        After online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize.
+        After consultation with Mr. Agrawal we optimized the prize-winning-logo for this year's summit as following:</p>
+    </div>
+    <div class="logo-container">
+      <img src="/assets/logo-about.png" alt="">
+    </div>
+    <hr class="horizontal-line">
+      
+  </section>
+
+  <section class="past-section">
+    <div class="past-section-container flex-past">
+      <h2>See the past StartUp Lab Summits</h2>
+      <hr>
+      <p>Take a look at the last two CC Global Summits which took place in Buenos Aires and in Warsaw.</p>
+      <img src="assets/2019.png" alt="">
+      <img src="assets/2017.png" alt="">
+    </div>
+  </section>
+
+  <!-- <section class="partners-section" id="partners">
+    <h2 class="partners-heading"><strong>Partners</strong></h2>
+    <hr/>
+    <div class="partners-flex">
+    <div class="partners-list">
+      <img class="partner-image" src="assets/partner1.png" alt=""/>
+    </div>
+    <div class="partners-list">
+      <img class="partner-image" src="assets/partner2.png" alt=""/>
+    </div>
+    <div class="partners-list">
+      <img class="partner-image" src="assets/partner3.png" alt=""/>
+    </div>
+    <div class="partners-list">
+      <img class="partner-image" src="assets/partner4.png" alt=""/>
+    </div>
+    <div class="partners-list">
+      <img class="partner-image" src="assets/partner5.png" alt=""/>
+    </div>
+  </div>
+  </section> -->
```

Comment:
- [ ] Delete all the commented code, remember that a clean code it's a happy code